src/streaming/Stream.js

```diff
@@ -768,6 +771,10 @@ function Stream(config) {
             let streamProcessor = streamProcessors[i];
             streamProcessor.updateStreamInfo(streamInfo);
             let mediaInfo = adapter.getMediaInfoForType(streamInfo, streamProcessor.getType());
+            if (!mediaInfo) {
+                // This can happen if AdaptationSet has been remove in refreshed MPD
+                break;
```

Comment:
Do we really need to call break here or is it sufficient to check mediaInfo? 
````
if(mediaInfo) {
abrController.updateTopQualityIndex(mediaInfo);
streamProcessor.addMediaInfo(mediaInfo, true);
}
````

Reply:
same, I can do it this way if you prefer

Reply:
Yes thanks